pkg/server/interceptors.go

```diff
@@ -29,20 +29,19 @@ func logOnErrorStreamInterceptor(srv interface{}, ss grpc.ServerStream, info *gr
 			err == io.EOF {
 			return err
 		}
-		zap.S().Errorf("%+v", err)
+		server.log.Sugar().Errorf("%+v", err)
```

Comment:
Not to change in this branch, but should we use the normal log rather than the sugared one?

Reply:
Tried to avoid figuring this out, but probably yes.

Reply:
It was a chance to ask, I saw it on Monday while working on something and it popped up here.